### DISCUSSION TITLE
WLJS Notebook 2.0 is out 🚀

### DISCUSSION THREAD

**Discussion by JerryI** [WLJSTeam/wljs-notebook#84] [Documentation]
<img width="1114" alt="light" src="https://github.com/JerryI/wolfram-js-frontend/assets/4111822/2ab4edf9-efab-414f-91e0-d9692ca62eeb">

It's been a long time, since the last release came out. WLJS Notebook and all supplementary plugins were rewritten from scratch in order to enforce modularity, to make maintainable code-base and more user-friendly UI.

There are some breaking changes in some APIs, which will be reflected soon in our documentation.

<hr /><em>This discussion was created from the release <a href='https://github.com/JerryI/wolfram-js-frontend/releases/tag/'>WLJS Notebook 2.0 is out 🚀</a>.</em>

**Comment by Dustingyd** [comment]
Excuse me, why is this happening? 

```
MacBook Air M1 
MacOS 13.4.1 
WLJS.Notebook-2.2.2-arm64.dmg
```

<img width="260" alt="image" src="https://github.com/JerryI/wolfram-js-frontend/assets/93420760/816cc95c-333e-4fb9-b5cc-41b39909769f">

**Reply by JerryI** [reply]
Hello @Dustingyd,
Unfortunately Apple make the rules more tight for M1/M2 users.

There is a workaround solution on how to fix this https://github.com/JerryI/wolfram-js-frontend/issues/61#issuecomment-1978511276 

You can move an App to your application folder and then apply the solution from the comment. 

Open the terminal, type the following command, and then press the return key:

`xattr -cr "<installation location>.app"`
The <installation location> should be replaced with the file path to where the app has been installed

**Reply by Dustingyd** [reply]
Thx for the solution and your great work!!

**Reply by JerryI** [reply]
I added a guide for [Apple Silicon to Docs page](https://jerryi.github.io/wljs-docs/frontend/Troubleshooting/Apple%20Silicon)

**Comment by Dustingyd** [comment]
> I added a guide for [Apple Silicon to Docs page](https://jerryi.github.io/wljs-docs/frontend/Troubleshooting/Apple%20Silicon)

Great!!! It works well in the browser now!

**Reply by JerryI** [reply]
@Dustingyd , Ok we both Apple developer subscription for a year ;D

https://github.com/JerryI/wolfram-js-frontend/releases/tag/2.2.6

Now it must be fine with installation
